DAO.sol

```diff
@@ -500,6 +500,26 @@ contract DAO is DAOInterface, Token, TokenSale {
 
         uint quorum = p.yea + p.nay;
 
+        // require 53% for callin newContract
+        if (_transactionData.length >= 4 && _transactionData[0] == 0x68
+            && _transactionData[1] == 0x37 && _transactionData[2] == 0xff
+            && _transactionData[3] == 0x1e
+            && quorum < minQuorum(actualBalance() + totalRewardToken)) {
+
+            if (!p.creator.send(p.proposalDeposit))
+                throw;
+
+            sumOfProposalDeposits -= p.proposalDeposit;
```

Comment:
abstract the following into a single internal function.
```js
sumOfProposalDeposits -= p.proposalDeposit;
// Since the voting deadline is over, close the proposal
p.open = false;
// Initiate event
ProposalTallied(_proposalID, _success, quorum);
```

Then use it here, and also again at the end of `executeProposal()`